test_scripts/API/AutoCompleteList/002_ATF_SetGlobalProperties_P_AutoCompleteList_WithOutParam.lua

```diff
@@ -0,0 +1,82 @@
+--------------------------------------------------------------------------------------------------------------
+-- Requirement summary:
```

Comment:
@Aleksandriys, please add to also to requirement summary next CRQ:
[SetGlobalProperties] SDL must transfer request to HMI in case "autoCompleteList" param omited in request from mobile app